docs/applications/create-commit-statuses-from-application-deployment-statuses.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Create Commit Statuses from Application Deployment Statuses
+intro: Outgoing Webhooks allow users to integrate Devopness to their current CI/CD Workflows. Learn how to create outgoing webhooks to update a commit status check to match your Devopness application deploy pipeline status.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+NOTE: Webhooks, for now, are an API only feature; so this post will guide you through the usage of our API to help you create an outgoing webhook.
+
+NOTE: cURL requests can be imported to API clients, by following the API client's instructions on the links below:
+
+- Postman: [Importing and exporting data: Importing with cURL commands](https://learning.postman.com/docs/getting-started/importing-and-exporting-data/#importing-with-curl-commands)
+- Insomnia: [Import and Export Data: Import Data](https://docs.insomnia.rest/insomnia/import-export-data#import-data)
+
+NOTE: Insomnia uses curly braces syntax for environment variables, to avoid errors you need to disable the nunjuncks template feature for the request body. Further instructions at [Insomnia's FAQ](https://docs.insomnia.rest/insomnia/faq#how-can-i-temporarily-disable-nunjucks-template)
+
+1. Take note of the `Application ID` (`<application_id>`) and `Deploy Pipeline ID` (`<pipeline_id>`) from the application which you want to watch the action statuses
+
+   - Follow the [Deploy Application using an Incoming Hook](/docs/applications/deploy-application-using-incoming-hook) guide for detailed instructions
+
+1. Take note of the `Repository Owner` (`<repo_owner>`) and `Repository Name` (`<repo_name>`) from the URL to the GitHub repository where the source code is hosted
```

Comment:
@thlmenezes the Devopness user may have you code base in another `source provider` and not only in `Github`

Reply:
Makes sense

Reply:
Instruction updated